examples/sld/main.js

```diff
@@ -0,0 +1,874 @@
+// This product includes color specifications and designs developed by Cynthia Brewer (http://colorbrewer.org/).
+var colorbrewer = {
+  YlGn: {
+    3: ['#f7fcb9', '#addd8e', '#31a354'],
+    4: ['#ffffcc', '#c2e699', '#78c679', '#238443'],
+    5: ['#ffffcc', '#c2e699', '#78c679', '#31a354', '#006837'],
+    6: ['#ffffcc', '#d9f0a3', '#addd8e', '#78c679', '#31a354',
+            '#006837'
+        ],
+    7: ['#ffffcc', '#d9f0a3', '#addd8e', '#78c679', '#41ab5d',
+            '#238443', '#005a32'
+        ],
+    8: ['#ffffe5', '#f7fcb9', '#d9f0a3', '#addd8e', '#78c679',
+            '#41ab5d', '#238443', '#005a32'
+        ],
+    9: ['#ffffe5', '#f7fcb9', '#d9f0a3', '#addd8e', '#78c679',
+            '#41ab5d', '#238443', '#006837', '#004529'
+        ]
```

Comment:
We are limiting the color count to 8.  Is there some virtue to keeping the entries for >= 9 in the `colorbrewer` object?